src/components/Nav.svelte

```diff
@@ -51,15 +73,28 @@
     padding: 1em 0.5em;
     display: block;
   }
+  
+  #login{
+    visibility:hidden; 
+    } 
+
+  #Welcome{
+  visibility:hidden; 
+  float: right;     
+  } 
+  
 </style>
 
 <nav>
   <ul>
     <li>
       <a class:selected={segment === undefined} href="/.">Home</a>
     </li>
```

Comment:
Unlike in `d10n`, you should be able to use [Svelte #if directives](https://svelte.dev/docs#if) to determine whether the login button or the welcome message should show up here, as opposed to using `visibility: hidden`. I might be wrong.